chrome-extension/test/configure.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.exit();
```

Comment:
Any reason this is still here? Unless I'm mistaken, this disables the entire test.